src/tree/utils/populateTree.ts

```diff
@@ -76,6 +82,13 @@ export const populateTree: Walker = (schema, parent, level, path, options): unde
     } else if (_isObject(node.properties)) {
       (treeNode as TreeListParentNode).children = [];
 
+      if (options?.mergeAllOf && (node.combiner === 'oneOf' || node.combiner === 'anyOf')) {
```

Comment:
Curious - when do we ever not merge all of? Is there a use case in our ecosystem?

Reply:
>  Is there a use case in our ecosystem?

I don't think so. IIRC we didn't want allOf for masking back in the day, but that's no longer relevant.

Reply:
Oh I'm saying that is there ever a reason to leave allOf in place? Versus merging it out.

Reply:
Yeah, I missed one word in the sentence above and the entire sense was changed.
I said we hadn't wanted allOf merging for masking some time ago, but now, allOf merging is supported by masking and enabled by default.
That being said, there doesn't seem to be any use case anymore.